rfcs/text/0002-type-ann-for-struct-properties-in-structs.md

```diff
@@ -0,0 +1,97 @@
+- Feature Name: type-annotations-for-struct-properties-in-structs
+- Start Date: TODO
+- RFC PR: (leave this empty)
+- Feature Commit(s): (leave this empty)
+
+# Summary
+
+Typecheck a value expression for a struct property in a struct definition with
+expected type information in the property.
+
```

Comment:
My knee-jerk reaction is that this summary is too specific and will be confusing for anyone not intimately familiar with the development story behind all of this (which shouldn't be relevant, IMHO). I.e., I think the fact that there is some (undocumented?) support for struct properties without type checking currently shouldn't be conveyed in the RFC title and summary. I think it would better communicate the intent to just say something like this RFC aims to "Provide initial support for type checking some struct properties" and then later on in the fine grained details we can discuss some of the nuance if that adds value. Just my two bits.

Reply:
Or perhaps I'm misreading it and imposing some of my own understanding of this work, but either way I still think having a simpler and more clear summary would add value.